src/app.js

```diff
@@ -15,10 +15,24 @@ export function createApp(todoStore, args) {
       ]);
       break;
     case 'add':
-      const validated = validateAddParams(params);
-      const added = add(todoStore, validated);
+      const validatedAdd = validateAddParams(params);
+      const added = add(todoStore, validatedAdd);
       display(['New Todo added:', format(added)])
       break;
+    case 'find-by-status':
+      const validatedStatus = validateFindByStatusParam(params);
+      const filteredTodos = findByStatus(todoStore, validatedStatus);
+      if(filteredTodos.length === 0) {
```

Comment:
Please move the array length checking to todo.js.

Reply:
moved it :)